RELEASE/scripts/autoscend/paths/wildfire.ash

```diff
@@ -451,6 +451,19 @@ boolean LX_wildfire_water()
 		LX_wildfire_hose($location[The Hidden Bowling Alley]);		//part of level 11 quest. potentially might want to go after NC instead
 	}
 	
+	if(in_hardcore() && !haveWarOutfit() && internalQuestStatus("questL12War") == 0)	//we need war outfit
+	{
+		abort("Due to tracking issues you need to manually acquire the necessary war outfit and run me again");
+		if(auto_warSide() == "fratboy")
+		{
+			LX_wildfire_hose($location[Frat House In Disguise]);
+		}
+		else
+		{
+			LX_wildfire_hose($location[Hippy Camp In Disguise]);
+		}
```

Comment:
is there any point in this being here if the abort exists right before it?

Reply:
I initially wrote the code to automate it, then when I saw that it is faulty due to tracking issues I added the abort which made the code unreachable.
I did not want to delete the code because hopefully this would be fixed upstream soon.

I should have commented out the code and added a comment explaining this

Reply:
```suggestion
// below is code for automation that is not functional due to mafia not tracking fire levels correctly.
//  https://github.com/Loathing-Associates-Scripting-Society/autoscend/issues/892#issuecomment-934059485
//		if(auto_warSide() == "fratboy")
//		{
//			LX_wildfire_hose($location[Frat House In Disguise]);
//		}
//		else
//		{
//			LX_wildfire_hose($location[Hippy Camp In Disguise]);
//		}
```